frontend/app/components/Session_/Inspector/AttrView.tsx

```diff
@@ -63,6 +63,7 @@ export default function AttrView({ attr, forceUpdateElement }: Props) {
 				value={ `${ attr.name }="${ attr.value }"` } 
 				commit={ commit }
 				className="ml-2"
+				style={{ maxWidth: '500px' }} 
```

Comment:
The component is repetitive, so it is better to use `className` for this. So better use CSS file or configure tailwind
https://tailwindcss.com/docs/max-width